### DISCUSSION TITLE
prompt between runs

### DISCUSSION THREAD

**Discussion by urbanjost** [fortran-lang/fpm#531] [Ideas]
Does anyone else wish there were a --prompt switch on the run subcommand so you could do something like
```bash
fpm run --example '*' --prompt
```
and it would pause and wait for a y/n response before executing?  When running a large number of executables I sometimes want to examine the output of each one before proceeding to the next, or maybe just say to run "*" but then decide on a case by case basis? I think it would be useful on the test subcommand as well.  

Alternatively, there could be a menu, or the globbing string could take a special form, although a question mark is already a legal globbing string.  I am thinking that instead of a new switch like --prompt you could do something like
```
fpm test '?'  
```
but that already represents a single-character executable name, albeit I suspect a very rare one; but I could picture someone calling their tests or examples  a,b,c,d, ... .

Does anyone know of a similar option on any other package managers?

**Comment by LKedward** [comment]
I haven't yet come across a need for this myself, however I can see why it could be useful. Perhaps this would be invoked by a more general `--interactive` flag which could have other uses to other subcommands? This might be something that you've already suggested in the past.
